src/actions/index.js

```diff
@@ -0,0 +1,21 @@
+import createRoom from '../socketHandlers/createRoom';
+import io from 'socket.io-client';
+const socket = io('localhost:9000/game');
```

Comment:
https://github.com/duyoji/oni-on-cheese/pull/56/files#r157937878

same as this.
And, we should write this code in one place. because client should have only one connection.